src/components/AddForm/AddForm.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './AddForm.css'
+import AddButton from "../AddButton/AddButton"
+import CancelButton from "../CancelButton/CancelButton"
+import AcceptButton from "../AcceptButton/AcceptButton"
+import InputField from "../InputField/InputField"
+import { useState } from 'react';
+
+function AddForm({ addTodo }) {
+
+    const [inputValue, setInputValue] = useState('')
+    const [activeInput, setActiveInput] = useState('inactive')
```

Comment:
if your state have only 2 variants, then it should be boolean instead of string.